### PR TITLE
submitting connector for certification

### PR DESCRIPTION
**Pull request by jhan-zaib**
SigningHub 1.1 version is already certified https://powerautomate.microsoft.com/en-us/connectors/details/shared_signinghub/signinghub/.  Now we made few updates in 1.2 version e.g. added an action and renamed an existing one.

---
#### Important: Our certification engineer, Srikanth, is currently on emergency leave. Please expect a delay with your PR. He will return January 31st, 2022. We apologize for any inconveniences.

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by jhan-zaib** [comment]
Hi @sriyen-msft, I have submitted connector 28 days ago but still not picked for reviewing process.  Kindly let me know how I can boost this process?

Thanks,
Jhan Zaib

File: certified-connectors/SigningHub/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+  "properties": {
+    "capabilities": [],
+	"publisher": "Ascertia",
+    "stackOwner": "Ascertia",
+    "connectionParameters": {
+      "token": {
+        "oAuthSettings": {
+          "clientId": "0123456789",
```
Comment (sriyen-msft) [inline certified-connectors/SigningHub/apiProperties.json:9]: @jhan-zaib You can just keep it as `"[[TEST]]" or `"[[DUMMY]"`.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/SigningHub/Readme.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+
+## SigningHub Connector
+SigningHub is a complete solution for document approval workflows, advanced digital signatures and document status tracking. It is designed to quickly optimize the way businesses deliver, review, approve and sign their business documents
+
+
+## Pre-requisites
+You will require a SigningHub account (https://www.signinghub.com/). You can free SignUp from https://web.signinghub.com/
```
[the remark this answers is not in the document]
Comment (jhan-zaib) [reply certified-connectors/SigningHub/Readme.md:7]: yes - you can free signup using direct link https://web.signinghub.com/Register?planId=30

File: certified-connectors/SigningHub/apiDefinition.swagger.json.bak [hunk 3]
```diff
@@ -0,0 +1,1935 @@
+{
```
[the remark this answers is not in the document]
Comment (jhan-zaib) [reply certified-connectors/SigningHub/apiDefinition.swagger.json.bak:1]: deleted .bak file

File: certified-connectors/SigningHub/apiProperties.json (hunk 1 above)
Comment (jhan-zaib) [reply certified-connectors/SigningHub/apiProperties.json:9]: added dummy text

File: certified-connectors/SigningHub/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,1936 @@
...
+        "description": "Get the SigningHub instance details",
+        "operationId": "AboutSigningHub",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "properties": {
+                "aboutSigningHub": {
+                  "description": "aboutSigningHub",
```
[the remark this answers is not in the document]
Comment (jhan-zaib) [reply certified-connectors/SigningHub/apiDefinition.swagger.json:32]: updated description

File: certified-connectors/SigningHub/Readme.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+
+## SigningHub Connector
+SigningHub is a complete solution for document approval workflows, advanced digital signatures and document status tracking. It is designed to quickly optimize the way businesses deliver, review, approve and sign their business documents
+
+
```
[the remark this answers is not in the document]
Comment (sriyen-msft) [reply certified-connectors/SigningHub/Readme.md:5]: @jhan-zaib Please add publisher section

File: certified-connectors/SigningHub/Readme.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sriyen-msft) [reply certified-connectors/SigningHub/Readme.md:7]: @jhan-zaib Please update the wording to `"You can get a free SignUp account from https://web.signinghub.com/ "`

File: certified-connectors/SigningHub/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,1936 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "SigningHub is a complete solution for document approval workflows, advanced digital signatures and document status tracking. It is designed to quickly optimize the way businesses deliver, review, approve and sign their business documents.",
+    "title": "SigningHub",
+    "version": "1.2.0",
+		"contact":{
```
Comment (sriyen-msft) [inline certified-connectors/SigningHub/apiDefinition.swagger.json:7]: @jhan-zaib Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by Amjed-Ayoub** [comment]
Dear partner,

We apologize for the delay you might be experiencing while we are working on a high volume of connector requests. Rest assured that we are doing everything we can to address this volume. We are taking measures like improving our processes and hiring additional team members to guarantee resolving your connector requests in a timely manner. 

Please let us know if you have any questions in the meantime.

**Comment by sriyen-msft** [comment]
@jhan-zaib  I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback (all the comments need to be addressed to review your PR). Please feel free to reach out with any questions you may have, Thanks.

**Comment by jhan-zaib** [comment]
@sriyen-msft I have fulfilled all the points mentioned in the last review and waiting for review completion. Let me know what's left at our side to do. Thanks

**Comment by jhan-zaib** [comment]
@sriyen-msft what's update on this? I believe all comments are responded properly.

File: certified-connectors/SigningHub/Readme.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+## SigningHub Connector
+SigningHub is a complete solution for document approval workflows, advanced digital signatures and document status tracking. It is designed to quickly optimize the way businesses deliver, review, approve and sign their business documents
+
+##Publisher
```
Comment (v-eisljami) [inline certified-connectors/SigningHub/Readme.md:5]: There are still some errors with Readme.md file. ## Publisher is a correct way of defining header. Please review the file using this [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md).

File: certified-connectors/SigningHub/apiDefinition.swagger.json [hunk 8]
```diff
@@ -0,0 +1,1936 @@
...
+                      "x-ms-visibility": "important"
+                    },
+                    "installation_name": {
+                      "description": "Installation name of SigningHub instance",
+                      "title": "Installation name",
+                      "type": "string",
+                      "x-ms-visibility": "important"
+                    },
+                    "patents": {
+                      "description": "This containc SigningHub US patent number",
```
Comment (v-eisljami) [inline certified-connectors/SigningHub/apiDefinition.swagger.json:53]: All descriptions must be free from grammatical errors. Please review all descriptions, summaries, etc.

Comment (jhan-zaib) [reply certified-connectors/SigningHub/apiDefinition.swagger.json]: removed spell and grammatical errors

**Comment by v-eisljami** [comment]
Hey @jhan-zaib. I hope you are doing well. There seems to be a problem with API Properties Validator. The error message is:
"##[error]@{level=Error; message=JSON does not match all schemas from 'allOf'. Invalid schema indexes: 0.; valuePath=line: 0 position: 0 path: 'properties.connectionParameters.token:TenantId.uiDefinition'; value=}". 
We cannot merge it unless all of the validations are met. Can you please review this and get back to us if you need any help.

**Comment by jhan-zaib** [comment]
Hi @v-eisljami,
You have checked there is only one error left in validating apiProperties.json which is new for me. So can you help me in this regard?

File: certified-connectors/SigningHub/apiProperties.json [hunk 9]
```diff
@@ -0,0 +1,51 @@
+{
+  "properties": {
+    "capabilities": [],
+	"publisher": "Ascertia",
```
Comment (sriyen-msft) [inline certified-connectors/SigningHub/apiProperties.json:4]: @jhan-zaib Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
> Hi @v-eisljami, You have checked there is only one error left in validating apiProperties.json which is new for me. So can you help me in this regard?

@jhan-zaib Sorry for the delay in reviewing your PR, there is known issue in the apiProperties.json file validator, please ignore it for now. Thanks

**Comment by sriyen-msft** [comment]
@jhan-zaib I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

**Comment by jhan-zaib** [comment]
Hi @sriyen-msft , we are looking for your review process completion. Its already in-process from last 5 months.

File: certified-connectors/SigningHub/apiDefinition.swagger.json (hunk 6 above)
Comment (sriyen-msft) [reply certified-connectors/SigningHub/apiDefinition.swagger.json:7]: @jhan-zaib Please fix the alignment of the properties in the swagger.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/SigningHub/apiDefinition.swagger.json (hunk 6 above)
Comment (jhan-zaib) [reply certified-connectors/SigningHub/apiDefinition.swagger.json:7]: Strange thing, this indentation and alignment changes are made various times but after pushing to GIT, no idea how indentation and alignment disturbed?

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**